interceptor/pkg/interceptor/pdinterceptor_test.go

```
@@ -224,13 +224,10 @@ func stringContains(s, substr string) bool {
 
 func TestOversizedRequestBodyIsRejected(t *testing.T) {
 	oversizedBody := bytes.Repeat([]byte("A"), 10*1024*1024) // 10 MiB
-	req := httptest.NewRequestWithContext(context.Background(), http.MethodPost, "/", bytes.NewReader(oversizedBody))
+	req := httptest.NewRequest(http.MethodPost, "/", bytes.NewReader(oversizedBody))
```

golangci-lint (2.12.2)

[error] net/http/httptest.NewRequest must not be called. use net/http/httptest.NewRequestWithContext

(noctx)